packages/compass-editor/src/editor.tsx

```diff
@@ -1537,7 +1468,7 @@ const MultilineEditor = React.forwardRef<EditorRef, MultilineEditorProps>(
         className={cx(
           multilineEditorContainerStyle,
           darkMode && multilineEditorContainerDarkModeStyle,
-          !!actions.length && multilineEditorContainerWithActionsStyle,
+          hasActions && multilineEditorContainerWithActionsStyle,
```

Comment:
The `actions.length` was actually always `> 0` (because we didn't trim the array from `false` values that could arise from the `copyable` or `formattable` being false.